docs/concepts/managed-resources.md

```diff
@@ -439,9 +459,12 @@ including Velero.
 
 [term-xrm]: terminology.md#crossplane-resource-model
 [composition]: composition.md
-[api-versioning]: https://kubernetes.io/docs/reference/using-api/api-overview/#api-versioning
+[api-versioning]:
+    https://kubernetes.io/docs/reference/using-api/api-overview/#api-versioning
 [velero]: https://velero.io/
 [api-reference]: ../api-docs/overview.md
 [provider]: providers.md
 [issue-727]: https://github.com/crossplane/crossplane/issues/727
 [issue-1143]: https://github.com/crossplane/crossplane/issues/1143
+[managed-api-patterns]:
+    https://github.com/crossplane/crossplane/blob/master/design/one-pager-managed-resource-api-design.md
```

Comment:
I think this may break these links

Reply:
Just checked that it doesn't

Reply:
Gotcha 👍🏻 Wasn't sure if it would break in site rendering.